spec/fixtures/service_tokens.yml

```diff
@@ -27,9 +27,23 @@
 
 permitted_top_group_token:
   layer: top_layer
-  name: Permitted
+  name: PermittedTopGroupToken
   description: MyText
-  token: PermittedToken
+  token: PermittedTopGroupToken
```

Comment:
Könnte es sein dass die Tests kaputt gegangen sind weil du das hier geändert hast? Falls es das ist, könnten dadurch auch Tests in den Wagons kaputt gehen.

Reply:
Sind schon tests im core deshalb kaputt gegangen aber merci für den Hinweis. Die Wagons hätte ich wohl vergessen

Reply:
Dann lass ich lieber den Namen wie er war, sonst gibts nur unnötig viel Aufwand